docs/installation/standalone.mdx

```diff
@@ -0,0 +1,417 @@
+---
+title: Standalone Installation
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Overview
+
+This documents will show you how to install KubeVela in standalone mode. There are two options:
+
+1. [Local machine](#local): Install in a local machine.
```

Comment:
```suggestion
1. [Local machine](#local): Install on a local machine based on Linux, MacOS or Windows operating system.
```

Reply:
Little tips here: the standard writing of "MacOS" is actually "macOS"